diversity-lunch-be/src/test/java/de/brockhausag/diversitylunchspringboot/meeting/controller/MeetingControllerTest.java

```diff
@@ -101,6 +101,32 @@ void testDeleteMeeting_serviceDeleteMeeting() {
         assertEquals(HttpStatus.OK, response.getStatusCode());
     }
 
+    @Test
+    void testDeleteMeeting_isNotDeleted_If_meetingProposalIsMatched() {
+        long id = 1L;
+        MeetingProposalEntity meetingProposalEntity1 = meetingTestdataFactory.entity();
+        meetingProposalEntity1.setMatched(true);
+
+        when(this.meetingService.checkIfMeetingProposalIsMatched(id)).thenReturn(true);
+
+        ResponseEntity<String> response = this.meetingController.deleteMeetingProposal(id);
+
+        assertEquals(HttpStatus.FORBIDDEN, response.getStatusCode());
+    }
+
+    @Test
+    void testDeleteMeeting_isDeleted_If_meetingProposalIsNotMatched() {
+        long id = 1L;
+        MeetingProposalEntity meetingProposalEntity1 = meetingTestdataFactory.entity();
+        meetingProposalEntity1.setMatched(true);
```

Comment:
Da es in Zeile 123 gemocked wird spielt das keine Rolle aber müsste das nicht auf false gesetzt werden ? Oder weil ihr es mocked gar nicht gesetzt werden?